### DISCUSSION TITLE
Follow comments

### DISCUSSION THREAD

**Discussion by bjoern-tantau** [Docile-Alligator/Infinity-For-Reddit#963] [Ideas]
This seems to be a newish feature in the official app. But apparently it is not available via the web interface. 

I would like to be notified of new comments made on a post or new answers made to a comment. I think one way to implement this would be by subscribing to the posts/comments rss feed.

**Comment by Wladefant** [comment]
Duplicate: #954 and #872

**Comment by Pjjlltt** [comment]
This would be a great addition to Infinity!

**Comment by craig872** [comment]
This is the one thing Infinity lacks that I still use the official Reddit app for.

**Comment by craig872** [comment]
Multiple instances of this request here on GH and also on the reddit forum.
Can we get an update if this is anywhere on the radar @Docile-Alligator ?

See
https://github.com/Docile-Alligator/Infinity-For-Reddit/issues/666
https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/954
https://www.reddit.com/r/Infinity_For_Reddit/comments/noocne/track_new_comments_follow_posts/

**Reply by TetradoToxinTTX** [reply]
+1 for this. 
Would like to see!
